daft/execution/physical_plan.py

```diff
@@ -271,6 +272,131 @@ def join(
                 return
 
 
+def broadcast_join(
+    broadcaster_plan: InProgressPhysicalPlan[PartitionT],
+    reciever_plan: InProgressPhysicalPlan[PartitionT],
```

Comment:
Nit naming: receiver_plan

Reply:
Ah weird, I thought that I fixed that but my find/replace must have missed it!